libsolidity/ast/Types.h

```diff
@@ -1263,18 +1295,21 @@ class FunctionType: public Type
 		strings const& _parameterTypes,
 		strings const& _returnParameterTypes,
 		Kind _kind,
-		bool _arbitraryParameters = false,
-		StateMutability _stateMutability = StateMutability::NonPayable
+		StateMutability _stateMutability = StateMutability::NonPayable,
+		Options _options = Options{false, false, false, false, false}
```

Comment:
Would this not work?
```suggestion
		Options _options = Options{}
```

Reply:
No, didn't work for me.

Reply:
weird, though... don't see why it *wouldn't* work...